src/yadism/observable_name.py

```diff
@@ -140,15 +120,13 @@ def is_heavylight(self):
     @property
     def is_composed(self):
         """Is it a composed flavor? i.e. total"""
-        return self.flavor == "total"
+        return "total" in self.flavor
```

Comment:
This one should be the exact same, since only `"total"` contains `"total"`. And the former one is more restrictive.

I would keep the old one.